inst/tutorials/grading-demo/grading-demo.Rmd

```diff
@@ -258,35 +254,90 @@ sqrt(log(2))
 grade_code("Don't worry, things will soon get harder.")
 ```
 
-* Now, throw an error message of `"boom"`
+## Grading exercise errors
 
-```{r grade_error, exercise = TRUE}
+Since `enable_gradethis()` sets `learnr::tutorial_options(exercise.error.checker = grade_learnr_error)`, when an exercise submission produces an evaluation error, intelligent feedback is provided via `grade_code()` (when a solution is provided). Try it yourself by submitting `mt` (R error, incorrect answer), then `cars` (No error, incorrect answer), then `mtcars` (correct answer) below:
```

Comment:
I think this needs to be updated to the new option name, i.e.

```suggestion
Since `enable_gradethis()` sets `learnr::tutorial_options(exercise.error.check.code = "gradethis::grade_code()")`, when an exercise submission produces an evaluation error, intelligent feedback is provided via `grade_code()` (when a solution is provided). Try it yourself by submitting `mt` (R error, incorrect answer), then `cars` (No error, incorrect answer), then `mtcars` (correct answer) below:
```

Reply:
Nice catch